projectDocs/issues/triage.md

```diff
@@ -0,0 +1,178 @@
+## Intent
+This page intends to outline some of the information that might be helpful to those trying to triage issues.
+
+Most of the issues raised on the NVDA GitHub repository fall into one of the following categories:
+
+- adding support to new applications
+- adding new features in NVDA
+- adding new features to support 3rd party applications better
+- bug reports
+
+Firstly we want to catch high priority issues and ensure that they are attended to first.
+This might be things like:
+
+- a crash of NVDA
+- synthesisers not working correctly
+- an error in an existing feature.
+
+Secondly, we want to ensure that there is enough information on the issue so that it can be well understood and work can start when it comes to the front of the queue. The sooner we do this, the more likely it is that we will get the information we need.
+Most of the information required is asked for in the Github issue templates, so this is a great place to start.
+
+## Check for duplicates
+Pick a few keywords and search the NVDA repository on Github. We can also check if there is already a pull request in the works that may fix this issue.
+
+## What kind of issue is it?
+Can we label this as a regression, a requested change of behaviour, or a requested new feature?
+
+### Regression
+The behaviour of the software has changed, unintentionally. This may mean that a feature has stopped working altogether or there may be an error sound, or perhaps a crash.
+
+### New features
+This is something new, that NVDA does not do yet.
+
+### Change of behaviour
+The issue describes "Currently NVDA does something, but I would like it to do something else instead."
+
+## Is there a work around?
+We may already know of a work around, any existing alternatives, or if there is any way to achieve the request already?
+
+## Information to collect.
+
+This is the kind of information that might help when investigating the issue further.
+
+### Regressions
+
+- How is the regression triggered? We may call this the steps to reproduce, perhaps even STR for short. What this means is, that we are looking for the set of steps to make the bug happen on our own system.
+- What happens / what is the actual behaviour? Some examples might be:
+	- a crash
+	- a freeze
+	- an error noise
+	- A log message
+	- If there was an error noise or log message, was there any unexpected behaviour aside from this? For example, did NVDA fail to report something it should have?
+- Even if it seems obvious, what should happen instead? 
+	- Its worth clarifying this with the user, it helps to make sure everyone is on the same page, and that we truly understand what the issue is about.
+- What version of NVDA was being used. Its good to get something like: stable, beta, rc, alpha. But much better to get the exact version of NVDA, retrieved from the NVDA menu by going to "Help" then "About". "alpha-28931,186a8d70"
+- In which version of NVDA did this work as expected?

```

Comment:
May be make a note that it is much more likely that an issue is processed sooner when this info is provided, because we can easier find out whether it's a regression or not. It happens quite often that someone reports an issue in alpha without testing in last stable, for example. I've caught myself doing that too.